lib/normalize-option.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var valueOrFunction = require('value-or-function');
+
+module.exports = function normalizeOption(types, defaultValue, value) {
+  if (value === undefined) {
+    return defaultValue;
+  }
+  if (typeof types === 'string') {
+    types = [types];
+  }
+  var appliedArguments = [].slice.call(arguments, 3);
+  for (var index in types) {
```

Comment:
this loop seems very heavy handed to be used in every scenario when it is only accounting for 2 cases (`sourcemaps` and `since`).  Might need to think on this a bit more.